data/crac-creation/crac-creator-csa-profiles/src/main/java/com/powsybl/openrao/data/craccreation/creator/csaprofile/craccreator/contingency/CsaProfileContingencyCreator.java

```diff
@@ -31,6 +32,8 @@
 import java.util.Optional;
 import java.util.Set;
 
+import static com.powsybl.openrao.data.craccreation.creator.csaprofile.craccreator.CsaProfileConstants.REQUEST_CONTINGENCY;
```

Comment:
For harmony sake, use classic import for all constants or static import for all constants. for example  you used REQUEST_CONTINGENCIES_NAME as classic import